engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -184,6 +185,7 @@ namespace dmGameSystem
         {
                 {"position", 0, 3, dmGraphics::TYPE_FLOAT, false},
                 {"texcoord0", 1, 2, dmGraphics::TYPE_FLOAT, false},
+                {"page_index", 2, 1, dmGraphics::TYPE_FLOAT, false},
```

Comment:
We can't use ints for attributes because ES2 doesn't allow it.. :'(

Reply:
Now that I think of it, we could probably do this in a better way by adding the page to the UV coordinate and do: 

float page = floor(uv.s);
uv -= page;
sample = textureArray(tex, vec3(uv, page));